operations/power_management/Power_On_and_Start_the_Management_Kubernetes_Cluster.md

```diff
@@ -203,16 +203,18 @@ Verify that the Lustre file system is available from the management cluster.
     2021-08-04 17:28:21,945 - INFO - sat.cli.bootsys.ceph - Ceph is not healthy: The following fatal Ceph health warnings were found: POOL_NO_REDUNDANCY
     ```
 
-    The particular Ceph health warning may vary. In this example, it is `POOL_NO_REDUNDANCY`. See
-    [Manage Ceph Services](../utility_storage/Manage_Ceph_Services.md) for Ceph troubleshooting
-    steps, which may include restarting Ceph services as described below for convenience.
+    The particular Ceph health warning may vary. In this example, it is `POOL_NO_REDUNDANCY`.
 
-    Verify that the Ceph services started.
+    If the warning is `PG_NOT_DEEP_SCRUBBED`, this alert should clear once Ceph deep scrubs of PGs
+    have completed, likely within an hour or less. See [Ceph Deep
```

Comment:
```suggestion
    have completed, likely within an hour or less, but is dependent on the amount of outstanding deep scrubs and the load on the ceph cluster. See [Ceph Deep
```

Reply:
I don't want a large customer to hold us to an hour or less on a 20 nodes ceph cluster that is busy.

Reply:
Fair enough. Should I remove the "hour or less" entirely? They should be able to see it progressing as the number of PGs in a scrubbing state decreases, right?

Reply:
I removed the hour or less part as well and pushed a fix.